src/ObservationDialog/Select.js

```diff
@@ -264,12 +267,52 @@ export const SelectOne = ({
   )
 }
 
-export const SelectMultiple = ({ value, label }: Props) => {
+export const SelectOnev2 = ({
+  id,
+  value,
+  label,
+  options,
+  placeholder,
+  onChange,
+  ...props
+}: Props) => {
+  const classes = useStyles()
   return (
-    <TextField
-      value={Array.isArray(value) ? value.join(', ') : value}
-      label={label}
-      disabled
+    <Autocomplete
+      id={id}
+      value={value}
+      onChange={(e, v) => onChange(v)}
+      options={options.map(op => (typeof op === 'object' ? op.label : op))}
+      renderInput={params =>
+        renderInput({ ...params, classes, label, placeholder })
+      }
+      {...props}
+    />
+  )
+}
+
+export const SelectMultiple = ({
+  id,
+  value,
+  label,
+  options,
+  placeholder,
+  onChange,
+  ...props
+}: Props) => {
+  const classes = useStyles()
+  return (
+    <Autocomplete
+      id={id}
+      multiple
+      freeSolo
```

Comment:
freeSolo can be an option passed through by the component that defaults to `true` -- this can be done in a later feature PR, though, if you don't have time

Reply:
For sure! Note that since the component unwraps the `{...props}` on L315, you can actually use the SelectMultiple component like `<SelectMultiple freeSolo={false} />` in order to disable freeSolo on demand.

The `Props` flow type needs to be improved in that regard in order to extend `Autocomplete`'s props. But functionally it works already.